flux/stdlib/influxdata/influxdb/rules.go

```diff
@@ -675,3 +643,203 @@ func (SortedPivotRule) Rewrite(ctx context.Context, pn plan.Node) (plan.Node, bo
 	}
 	return pn, false, nil
 }
+
+//
+// Push Down of window aggregates.
+// ReadRangePhys |> window |> { min, max, mean, count, sum }
+//
+type PushDownWindowAggregateRule struct{}
+
+func (PushDownWindowAggregateRule) Name() string {
+	return "PushDownWindowAggregateRule"
+}
+
+var windowPushableAggs = []plan.ProcedureKind{
+	universe.CountKind,
+	universe.SumKind,
+	universe.MinKind,
+	universe.MaxKind,
+	universe.MeanKind,
+	universe.FirstKind,
+	universe.LastKind,
+}
+
+func (rule PushDownWindowAggregateRule) Pattern() plan.Pattern {
+	return plan.OneOf(windowPushableAggs,
+		plan.Pat(universe.WindowKind, plan.Pat(ReadRangePhysKind)))
+}
+
+func canPushWindowedAggregate(ctx context.Context, fnNode plan.Node) bool {
+	// Check the aggregate function spec. Require the operation on _value
+	// and check the feature flag associated with the aggregate function.
```

Comment:
comment is wrong about feature flags?

Reply:
Yeah this is outdated in IDPE since most arent flagged anymore. Here's the list for reference https://docs.google.com/spreadsheets/d/1yNWxrVoL9AEUVdoKVe9ZLWmqdpaSfYtxO2_lvxSk-Fw/edit#gid=0